Source/ExcelDna.AddIn.Tasks/ExcelDna.AddIn.Tasks.csproj

```diff
@@ -1,72 +1,39 @@
+    <Version>1.1.0.0</Version>
+    <FileVersion>1.2.2.0</FileVersion>
+    <InformationalVersion>1.2.2</InformationalVersion>
+    
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\assets\exceldna.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkProfile />
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
+
   </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ExcelDna.Integration\ExcelDna.Integration.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="EnvDTE" Version="8.0.2" />
+    <PackageReference Include="NUnit" Version="*" />
+  </ItemGroup>
+  
   <ItemGroup>
-    <Reference Include="envdte, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <EmbedInteropTypes>True</EmbedInteropTypes>
-    </Reference>
     <Reference Include="Microsoft.Build.Framework" />
     <Reference Include="Microsoft.Build.Utilities.v4.0" />
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Management" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AbstractTask.cs" />
-    <Compile Include="BuildItemSpec.cs" />
-    <Compile Include="Logging\BuildLogger.cs" />
-    <Compile Include="Logging\IBuildLogger.cs" />
-    <Compile Include="SetDebuggerOptions.cs" />
-    <Compile Include="Utils\MessageFilter.cs" />
-    <Compile Include="Utils\ExcelDnaProject.cs" />
-    <Compile Include="Utils\IDevToolsEnvironment.cs" />
-    <Compile Include="Utils\IExcelDnaProject.cs" />
-    <None Include="Properties\ExcelDna.AddIn.Tasks.targets" />
-    <Compile Include="CleanExcelAddIn.cs" />
-    <Compile Include="BuildTaskCommon.cs" />
-    <Compile Include="CreateExcelAddIn.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Utils\Bitness.cs" />
-    <Compile Include="Utils\DevToolsEnvironment.cs" />
-    <Compile Include="Utils\ExcelDetector.cs" />
-    <Compile Include="Utils\ExcelDnaPhysicalFileSystem.cs" />
-    <Compile Include="Utils\IExcelDetector.cs" />
-    <Compile Include="Utils\IExcelDnaFileSystem.cs" />
-  </ItemGroup>
+
+  <!-- TODO: Not sure what to do here -->
```

Comment:
Is this comment about the links to the `.props` and `.targets` files? Is yes, then they look good to me. I created these links just to have a shortcut to edit these files directly from Visual Studio.

If the comment is about all the `<Compile ...>` that were removed, they also look good.

(_for later_) Now that this project is converted to SDK-style, we could prob. use MSBuild to pack and get rid of the `.nuspec` (perhaps even `dotnet pack` would work) so something we can consider for a future PR - in which cases the `.props` and `.targets` would need a few more attributes to be included in the package.